packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -280,6 +280,35 @@ class HelpButton extends StatelessWidget {
   }
 }
 
+class ExpandAllButton extends StatelessWidget {
+  const ExpandAllButton({Key key, @required this.onPressed}) : super(key: key);
+
+  final VoidCallback onPressed;
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
nit: =>

Not sure what we like to do here.

Reply:
if it's more than a one line return, we should use brackets.